vector/src/main/res/values/strings.xml

```diff
@@ -1058,7 +1068,7 @@
 
     <!-- notification statuses -->
     <string name="notification_off">Off</string>
-    <string name="notification_on">On</string>
+    <string name="notification_silent">Silent</string>
```

Comment:
the string has to be renamed also in all other language files (and lint will be happy)

Reply:
I removed the notification_on from all language files as the translation is now incorrect

Reply:
ok